src/e2e/puppeteer/__tests__/drag-and-drop.ts

```diff
@@ -173,6 +173,27 @@ describe('drag', () => {
     const image = await screenshot()
     expect(image).toMatchImageSnapshot()
   })
+
+  it('drop hover after first thought of column one', async () => {
+    await paste(`
+        - x
+        - a
+          - =view
+            - Table
+          - =pin
+            - true
+          - b
+            - c
+          - d
+            - e
+      `)
+
+    await clickThought('x')
+    await dragAndDropThought('x', 'd', { position: 'before' })
+
+    const image = await screenshot()
```

Comment:
Let's set `em.testFlags.simulateDrop`. It will allow the test to cover the exact size and position of the drop target.

Reply:
I'm not sure I understand, should the code below `await simulateDragAndDrop({ drop: true })` removed, is what you are saying?

Reply:
I'm not sure I understand, should the code below await simulateDragAndDrop({ drop: true }) removed, is what you are saying?